tests/throw-close-test.toit

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import expect show *
+import watchdog show WatchdogServiceClient
+
+import .util
+
+main:
+  run-test: |client ms system-dog| test-recover client ms system-dog
+
+// If a watchdog gets lost/closed, another watchdog with the same name can take over.
+test-recover client/WatchdogServiceClient ms/int system-dog/FakeSystemWatchdog:
```

Comment:
What is this recovering? I don't understand the method name or the comment?

Reply:
bad copy/paste from existing test...
done.